app/controllers/api/v1/event_procedures_controller.rb

```diff
@@ -66,6 +71,14 @@ def event_procedure_params
       def event_procedure
         @event_procedure ||= EventProcedures::Find.result(id: params[:id]).event_procedure
       end
+
+      def total_amount_cents
```

Comment:
looks like this should not be here, maybe this object can be returned from some service/action